test_delete.py

```
@@ -1,13 +1,8 @@
 import pytest
-from unittest.mock import patch, mock_open, call
-from delete import (
-    validate_args,
-    read_user_ids,
-    get_base_url,
-    get_access_token,
-    delete_user
-)
-import requests
+from unittest.mock import patch, mock_open
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)